src/ansi-c/c_typecast.cpp

```diff
@@ -55,14 +55,14 @@ bool c_implicit_typecast_arithmetic(
   return !c_typecast.errors.empty();
 }
 
-bool is_void_pointer(const typet &type)
+bool has_a_void_pointer(const typet &type)
```

Comment:
I'm not sure what these front-end changes have to do with the description of the patch.

Reply:
I also introduced another function somewhere in the patch called ```is_void_pointer``` but with other semantics.

The new function checks, that type.id is pointer and subtype.id is void. The older front-end function checks that recursively on subtype. Therefore, I thought renaming the old function is the better choice to resolve the name conflict.

Reply:
OK...